README.md

```diff
@@ -40,7 +40,7 @@ Define the following requirement in your `composer.json` file:
 ```json
 {
   "require": {
-    "imgix/imgix-php": "dev-master"
+    "imgix/imgix-php": "dev-main"
```

Comment:
Just thinking out loud, this worries me that changing the base branch name could have negative effects for users. However, it looks like both tags are [available online](https://packagist.org/packages/imgix/imgix-php) -- thanks to our auto-updating hooks. 

Because of that, I _think_ it's ok to move forward with this now, but it would be worth modifying this install step with something like what is described in #21